docs/testnet/Introduction.md

```diff
@@ -1,7 +1,7 @@
 # Introducing Obscuro Testnet
 The Obscuro Testnet is the number 1 destination for application developers, builders and node operators who want to migrate their Ethereum smart contracts to a privacy preserving Layer 2. With Obscuro Testnet you can familiarise yourselves with deploying apps, running an Obscuro node or simply understand more about how Obscuro works.
 
-The Testnet release schedule has been intentionally designed to make new features available as soon as possible. This means you can expect frequent updates which will be communicated [here](https://github.com/obscuronet/obscuro-project/wiki/Testnet-Changelog). The trade-off is that in the early days Testnet will have some sharp edges and unexpected surprises. More specifically, whilst Testnet is in its infancy:
+The Testnet release schedule has been intentionally designed to make new features available as soon as possible. This means you can expect frequent updates which will be communicated [here](https://docs.obscu.ro/testnet/changelog.html). The trade-off is that in the early days Testnet will have some sharp edges and unexpected surprises. More specifically, whilst Testnet is in its infancy:
```

Comment:
Should the change log be updated as it just says preview release released in July 2022?

Reply:
Yes. Done